src/support/README.md

```diff
@@ -0,0 +1,185 @@
+# Microsoft Azure CLI 'support' Extension #
+
+## How to use ##
+
+Install this extension using the below CLI command
+
+```python
+az extension add --name support
+```
+
+## Included Commands ##
+
+### *"Services"* commands ###
+
+* #### List Services ####
+
+    *Examples:*
+
+    ```python
+    az support services list
+    ```
+
+* #### Get Single Service ####
+
+    *Examples:*
+
+    ```python
+    az support services show --service-name "ServiceNameGuid"
+    ```
+
+### *"Problem-Classifications"* commands ###
+
+* #### List Problem Classifications ####
+
+    *Examples:*
+
+    ```python
+    az support services problem-classifications list --service-name "ServiceNameGuid"
+    ```
+
+* #### Get Single Problem Classification ####
+
+    *Examples:*
+
+    ```python
+    az support services problem-classifications --service-name "ServiceNameGuid" show --problem-classification-name "ProblemClassificationNameGuid"
+    ```
+
+### *"Tickets"* commands ###
+
+* #### List Tickets ####
+
+    *Examples:*
+
+    ```python
+    # Vanilla variant
+    az support tickets list
+
+    # Tickets created after 1st January, 2020 that are still open (unresolved)
+    az support tickets list --filters "CreatedDate ge 2020-01-01 and Status eq 'Open'"
+    ```
+
+* #### Get Single Ticket ####
+
+    *Examples:*
+
+    ```python
+    az support tickets show --ticket-name "TicketCreatedFromPythonCLI"
+    ```
+
+* #### Update Ticket ####
+
+    *Examples:*
+
+    ```python
+    # Update Severity
+    az support tickets update --ticket-name "TicketCreatedFromPythonCLI" --severity "moderate"
+
+    # Update one or more Contact Details properties
+    az support tickets update --ticket-name "TicketCreatedFromPythonCLI" \
+        --contact-additional-emails "xyz@contoso.com" "devs@contoso.com" \
```

Comment:
Markdown syntax error I guess. 
![image](https://user-images.githubusercontent.com/14357159/72587060-dbf88e80-392e-11ea-97a5-bb4c7e3e4b51.png)

Reply:
yes I noticed. Not sure why it is rendered like this. Looks like a bug in GitHub. Same md file is correctly rendered in the repo when opened. Will remove python syntax highlight tag which seems to fix the problem.
![image](https://user-images.githubusercontent.com/13969733/72589253-8e890c80-38af-11ea-9046-e153a6d02599.png)